### PR TITLE
 [Refactor]: CDAT Migration Phase 3: testing and documentation update

### LINKED ISSUE #843: [Refactor]: CDAT Migration Phase 3: testing, documentation update, prepare for new release
### Is your feature request related to a problem?

This is the final phase for the CDAT Migration work. It involves testing, cleaning up docs, and preparing the codebase for a new release.

## 1. Examples
In the root `examples/` directory, there are example run scripts that use the legacy way of configuring and running E3SM diagnostics. They should all be refactored to reflect the latest changes.

### Tasks

- [ ] Make sure `examples` scripts work and produce close/same results as `main` 
  - [ ] `ex1`
  - [ ] `ex2`
  - [ ] `ex3`
  - [ ] `ex4`
  - [ ] `ex5`
  - [ ] `ex6`
  - [ ] `ex7`
- [ ] Update `/tutorials` as needed 
- [ ] Update `/tutorials/2024` as needed
- [ ] Delete all `run_...py` scripts (only keep `run_v2_9_0_all_sets_E3SM_machines.py`

### Tasks

- [ ] Update documentation to reflect any changes in the refactored codebase

### Tasks

- [ ] Update changelog
- [ ] Remove unused modules that contain CDAT dependencies
  - [ ] `general.py`
  - [ ] `dataset.py`
- [ ] Remove CDAT dependencies from conda-env `.yml` files
- [ ] Remove any lingering CDAT references (e.g., docs, docstrings, etc.)
- [ ] Bump to v3.0.0

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by tomvothecoder**
## Description

- Closes #843

## 1. Examples
In the root `examples/` directory, there are example run scripts that use the legacy way of configuring and running E3SM diagnostics. They should all be refactored to reflect the latest changes.

### Tasks

- [x] Make sure `examples` scripts work and produce close/same results as `main`
  - [x] `ex1`
  - [x] `ex2`
  - [x] `ex3`
  - [x] `ex4`
  - [x] `ex5` -- ~~a few variables failed due to [bug](https://github.com/E3SM-Project/e3sm_diags/pull/846#issuecomment-2332830874)~~ FIXED
  - [x] `ex6` -- ~~a few variables failed due to [bug](https://github.com/E3SM-Project/e3sm_diags/pull/846#issuecomment-2332830874)~~ FIXED 
  - [x] `ex7`
- [x] Fix incorrect time coordinate for obs dataset affecting `ex5` and `ex6` variable ([comment](https://github.com/E3SM-Project/e3sm_diags/pull/846#issuecomment-2332830874))
- [x] Copy fixed obs dataset from NERSC to LCRC -- in progress (@chengzhuzhang)
  - `/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`

### Tasks
- [x] Test run script with `model_vs_model` run
  - [x] Replace `get_ref_climo_dataset()` in all sets accept `lat_lon`
  - 124/128 variables within rtol
  - 4/128 not within rtol, but this is due to a bug on `main` (#797)
  - Some files are missing on `main` (refer to #854)
- [x] Re-run `model_vs_obs` with above change and perform regression 
  - After omitting specific files due to known issues (e.g., nan mismatch due to regridding):
    - 1213/1215 variables within rtol
    - 2/1215 variables not within rtol due to <=8/64800 elements mismatching (0.04% and 1.2% max rel diff) -- **not a concern**

### Tasks
- [x] Update `/tutorials` as needed -- none needed
- [x] Update `/tutorials/2024` as needed -- none needed
- [x] Delete all `run_...py` scripts (only keep `run_v2_9_0_all_sets_E3SM_machines.py`
- [x] Rename `run_v2_9_0_all_sets_E3SM_machines.py` to `run_all_sets_E3SM_machines.py`
- [x] Update documentation to reflect any changes in the refactored codebase
  - [x] CDP references 
 
## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

If applicable:

- [ ] New and existing unit tests pass with my changes (locally and CI/CD build)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have noted that this is a breaking change for a major release (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by tomvothecoder** [comment]
@chengzhuzhang The following variables failed in `ex5.py` and `ex6.py`:

```python
['/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_9.4_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_9.4_ISCCP-ANN-global_test.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_ISCCP-ANN-global_test.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU9.4_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex5_model_to_obs/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU9.4_ISCCP-ANN-global_test.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_9.4_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_9.4_ISCCP-ANN-global_test.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU1.3_ISCCP-ANN-global_test.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU9.4_ISCCP-ANN-global_ref.nc',
 '/global/cfs/cdirs/e3sm/www/cdat-migration-fy24/examples-dev/ex6_zonal_mean_2d_and_lat_lon_demo/lat_lon/Cloud ISCCP/ISCCPCOSP-CLDTOT_TAU9.4_ISCCP-ANN-global_test.nc']
```

### Root Cause
 I found that the time attributes for this file is bad, which breaks decoding (`decode_times=True`) with xCDAT/Xarray:
`/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`

### Checking time attributes (with `decode_times=False`)
```python
<xarray.DataArray 'time' (time: 1)> Size: 4B
array([150.5], dtype=float32)
Coordinates:
  * time     (time) float32 4B 150.5
Attributes:
    long_name:  Time
    units:      months since 1983-06
```

### Breaks with `decode_times=True`
```python
import xarray as xr
import xcdat as xc

filepath = "/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc"

ds_xc = xc.open_dataset(filepath)
# ValueError: Non-integer years and months are ambiguous and not currently supported.

ds_xr = xr.open_dataset(filepath)
# ValueError: Failed to decode variable 'time': unable to decode time units 'months since 1983-06' # with 'the default calendar'. Try opening your dataset with decode_times=False or installing
# cftime if it is not installed.
```

### Workaround 
This file can be opened with `decode_times=False` (which is why it worked in the CDAT codebase), but that means modifying the Xarray code to accommodate this specific dataset. I'm not a fan of logic to accommodate bad data. 

Unless we can get the time coordinates fixed, this might be the only other option. Happy to hear your thoughts.

- https://github.com/E3SM-Project/e3sm_diags/blob/9be8cea367fca31501be9c159fd64fda45fb5e28/e3sm_diags/driver/utils/dataset_xr.py#L310
- https://github.com/E3SM-Project/e3sm_diags/blob/9be8cea367fca31501be9c159fd64fda45fb5e28/e3sm_diags/driver/utils/dataset_xr.py#L442
- https://github.com/E3SM-Project/e3sm_diags/blob/9be8cea367fca31501be9c159fd64fda45fb5e28/e3sm_diags/driver/utils/dataset_xr.py#L490-L503

**Comment by tomvothecoder** [comment]
To correct myself, xCDAT **can** handle a missing "calendar" attribute by defaulting it to `standard` (Example: `2024-09-09 11:55:22,114 [WARNING]: dataset.py(decode_time:360) >> 'time' does not have a calendar attribute set. Defaulting to CF 'standard' calendar.`)

> ```python
> # ValueError: Non-integer years and months are ambiguous and not currently supported.
> ```

### Root cause of this issue:
Related dataset:  `/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`

**This issue is that this dataset uses a time scalar float of 150.5, which represents the middle of the month.** xCDAT's decoding logic uses the `relativedelta` module, which expects the time coordinates to be placed at the start of each the month (e.g., 150, 151). As a result, the `relativedelta` module raises `ValueError: Non-integer years and months are ambiguous and not currently supported.`


### MVCE 
```python
from datetime import datetime
from dateutil import parser
from dateutil import relativedelta as rd

import numpy as np

flat_offsets = np.array([150.5])
ref_date = "1983-06"
units_type = "months"

# 
ref_datetime: datetime = parser.parse(ref_date, default=datetime(2000, 1, 1))

# ValueError: Non-integer years and months are ambiguous and not currently supported.
times = np.array(
    [
        ref_datetime + rd.relativedelta(**{units_type: offset})
        for offset in flat_offsets
    ],
    dtype="object",
)
```

### Possible workarounds
1. Update dataset to replace scalar value 150.5 (representing 12/15/1995 or 12/16/1995 I think) with 150.0 (representing 12/01/1995)
 -- Maybe low risk because it is a single time coordinate and it is still within the same month of the climatology

**Comment by chengzhuzhang** [comment]
> ### Possible workarounds
> 
>     1. Update dataset to replace scalar value 150.5 (representing 12/15/1995 or 12/16/1995 I think) with 150.0 (representing 12/01/1995)
>        
>        * Maybe low risk because it is a single time coordinate and it is still within the same month of the climatology. I don't think this time coordinate is used downstream?

Hi Tom, I think the fractional month could be ambiguous and that's why `relativedelta` does not support that. I think the workaround you proposed would be fine in this case. I don't think the time coordinate is used later.

**Comment by tomvothecoder** [comment]
Hey @chengzhuzhang, the file with the fixed time coordinate that needs to be copied from NERSC to LCRC is `/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`. 

I created a backup of this file in the same directory on NERSC just in case. The backup has `.bak` appended to the filename.

**Comment by tomvothecoder** [comment]
### Findings for the  "Test run script with model_vs_model run" task

`main`
  - [Viewer link](https://portal.nersc.gov/project/e3sm/cdat-migration-fy24/main/v2_9_0_all_sets_model_vs_model/viewer/)
  - 128 `.nc` files  (not including diffs)
  - 143 `.png` files (not including diffs)
  - Sets: `area_mean_time_series`, `arm_diags`, `lat_lon`, `mp_partition`

`dev` 
  - [Viewer link](https://portal.nersc.gov/project/e3sm/cdat-migration-fy24/843-migration-phase3/viewer/)
  - 1040 `.nc` files` (not including diffs)
  - 635 `.png` files (not including diffs)
  - Sets: `annual_cycle_zonal_mean`, `area_mean_time_series`, `arm_diags`, `cosp_histogram`, `lat_lon`, `meridional_mean_2d`, `mp_partition`, `polar`, `zonal_mean_2d`, `zonal_mean_2d_stratosphere`, `zonal_mean_xy` 

### Why is `main` producing less files?

In the `main` [log file](https://github.com/E3SM-Project/e3sm_diags/blob/migration-phase3/auxiliary_tools/cdat_regression_testing/843-migration-phase3/run-script-model-vs-model/24-09-24-main-log.txt), the specific errors that I think are preventing the generation of the remaining files are:
```python
OSError: No file found for  and JJA in /global/cfs/cdirs/e3sm/diagnostics/observations/Atm/climatology
OSError: No file found for  and SON in /global/cfs/cdirs/e3sm/diagnostics/observations/Atm/climatology
OSError: No file found for  and MAM in /global/cfs/cdirs/e3sm/diagnostics/observations/Atm/climatology
OSError: No file found for  and DJF in /global/cfs/cdirs/e3sm/diagnostics/observations/Atm/climatology
```

### What I think is happening
- Dev branch -- `dataset_xr.py` has `get_ref_climo()` method that is being used for most/all sets. This method has a `try and except` statement that catches the `IOError` from `_get_climo_filepath()`, allowing sets to continue to run as a model-only run by substituting the test data for the reference data.
  - https://github.com/E3SM-Project/e3sm_diags/blob/9be8cea367fca31501be9c159fd64fda45fb5e28/e3sm_diags/driver/utils/dataset_xr.py#L318-L369
- Main branch -- uses `get_climo_variable()`  and only `lat_lon` set has the try and except for model-only runs. This means all other sets will fail, leading to less files being produced compared to dev.
  - https://github.com/E3SM-Project/e3sm_diags/blob/9be8cea367fca31501be9c159fd64fda45fb5e28/e3sm_diags/driver/utils/dataset.py#L120-L173
  - `lat_lon` code: https://github.com/E3SM-Project/e3sm_diags/blob/2b303dfdf1a83b115a8f372a81bf2f6b3bfbbc68/e3sm_diags/driver/lat_lon_driver.py#L149-L156 

### Options
1. Update logic on `get_ref_climo_dataset()` to perform model-only run for `lat_lon` if reference climo file is not found
2. Keep things as is if we want to perform model-only runs -- note, this might lead to slower overall runtime compared to main since more sets and variables are being processed

**Comment by tomvothecoder** [comment]
Hey @chengzhuzhang, can you provide your thoughts to my above [comment](https://github.com/E3SM-Project/e3sm_diags/pull/846#issuecomment-2372196528)?

**Comment by chengzhuzhang** [comment]
> Hey @chengzhuzhang, can you provide your thoughts to my above [comment](https://github.com/E3SM-Project/e3sm_diags/pull/846#issuecomment-2372196528)?

Hi @tomvothecoder I'm voting option 1 as the best. It can be a good opportunity to also address this relevant issue? https://github.com/E3SM-Project/e3sm_diags/issues/823 .

**Comment by chengzhuzhang** [comment]
> Hey @chengzhuzhang, the file with the fixed time coordinate that needs to be copied from NERSC to LCRC is `/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`.
> 
> I created a backup of this file in the same directory on NERSC just in case. The backup has `.bak` appended to the filename.

@tomvothecoder I have copied the data to LCRC input data serve. We can run mache to sync data to all machines once new e3sm_unified version is deployed.

**Comment by tomvothecoder** [comment]
> > Hey @chengzhuzhang, the file with the fixed time coordinate that needs to be copied from NERSC to LCRC is `/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology/ISCCPCOSP/ISCCPCOSP_ANN_climo.nc`.
> > I created a backup of this file in the same directory on NERSC just in case. The backup has `.bak` appended to the filename.
> 
> @tomvothecoder I have copied the data to LCRC input data server. We can run mache to sync data to all machines once new e3sm_unified version is deployed.

I found a bug with that file where the variable values were replaced with 0.0 for some reason. I recreated the file on NERSC.

Can you point me to the path and I will copy it over again?

**Comment by chengzhuzhang** [comment]
> Can you point me to the path and I will copy it over again?

The LCRC path is  `/lcrc/group/e3sm/public_html/diagnostics/observations/Atm/climatology/ISCCPCOSP`. Thank you for taking care of this @tomvothecoder

**Comment by tomvothecoder** [comment]
> > Can you point me to the path and I will copy it over again?
> 
> The LCRC path is `/lcrc/group/e3sm/public_html/diagnostics/observations/Atm/climatology/ISCCPCOSP`. Thank you for taking care of this @tomvothecoder

I can't overwrite the file due to permissions and different group access.

```python
-rw-r--r--  1 ac.zhang40 cels    2238458 Sep  9 17:12 ISCCPCOSP_ANN_climo.nc
-rwxr-xr-x  1 ac.zhang40 E3SM    2162972 Nov  5  2019 ISCCPCOSP_ANN_climo.nc.bak
-rwxr-xr-x  1 ac.zhang40 E3SM    2162972 Nov  5  2019 ISCCPCOSP_DJF_climo.nc
-rwxr-xr-x  1 ac.zhang40 E3SM    2162972 Nov  5  2019 ISCCPCOSP_JJA_climo.nc
-rwxr-xr-x  1 ac.zhang40 E3SM    2162972 Nov  5  2019 ISCCPCOSP_MAM_climo.nc
-rwxr-xr-x  1 ac.zhang40 E3SM    2162972 Nov  5  2019 ISCCPCOSP_SON_climo.nc
```

Whenever you have time, can you move the latest fixed version of this file from NERSC? Thanks!

File: e3sm_diags/driver/lat_lon_driver.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from __future__ import annotations
```
Comment (tomvothecoder) [inline e3sm_diags/driver/lat_lon_driver.py:1]: A file of interest for review

**Review by tomvothecoder — COMMENTED** [review]
Regression test results
- `examples` scripts -- all run, within rtol 
- `model_vs_model` -- 124/128 variables within rtol, 4/128 not (but okay due to bug on `main` -> refer to [PR description](https://github.com/E3SM-Project/e3sm_diags/pull/846#issue-2508957170)
- `model_vs_obs` -- 1213/1215 variables within rtol, 2/1215 not within rtol due to <=8/64800 elements mismatching (0.04% and 1.2% max rel diff) -- not a concern

Summary of Changes

e3sm_diags/driver/
- `annual_cycle_zonal_mean_driver.py`, `cosp_histogram_driver.py`, `meridional_mean_2d_driver.py`, `polar_driver.py`, `zonal_mean_2d_driver.py`, `zonal_mean_xy_driver.py`  -- replaced `get_ref_climo_dataset()` with `get_climo_dataset()`
- `lat_lon_driver.py`  
  - Add `_run_diags_2d_model_only()` and `_run_diags_3d_model_only()` functions (when `ds_ref` reference dataset is `None`)  
  - Add `_get_ref_climo_dataset()` -- now returns `None` if reference dataset is not passed instead of setting `ds_ref = ds_test`
  - Refactor `_create_metrics_dict()`
    - Use `DEFAULT_METRICS_VALUE` (999.99) as default value for missing metrics to support viewer, otherwise it will break if `None`  

`driver/utils/dataset_xr.py`
- Remove `get_ref_climo_dataset()` since it is only used by `lat_lon_driver.py`
- Fix `_subset_vars_and_load()` to keep `"area"` variable

Run scripts
- Delete all outdated `run_....py` scripts
- Rename `run_v_2_9_0_all_sets_E3SM_machines.py` to `run_all_sets_E3SM_machine.py` -- the filename can be version agonistic and updated based on the latest version

Update `complete_run.py`
- Port over functions from `run_v2_6_0_all_sets.py` needed in this module

File: e3sm_diags/driver/zonal_mean_xy_driver.py [hunk 2]
```diff
@@ -72,8 +72,8 @@ def run_diag(parameter: CoreParameter) -> CoreParameter:
             parameter._set_name_yrs_attrs(test_ds, ref_ds, season)
 
             ds_test = test_ds.get_climo_dataset(var_key, season)
-            # TODO consider to refactor the behavior of get_ref_climo_dataset
-            ds_ref = ref_ds.get_ref_climo_dataset(var_key, season, ds_test)
+            # TODO consider to refactor the behavior of get_climo_dataset
```
Comment (tomvothecoder) [inline e3sm_diags/driver/zonal_mean_xy_driver.py:75]:
```suggestion
```

File: e3sm_diags/driver/lat_lon_driver.py [hunk 3]
```diff
@@ -265,10 +440,100 @@ def _run_diags_3d(
             )
 
 
+def _get_ref_climo_dataset(
+    dataset: Dataset, var_key: str, season: ClimoFreq
+) -> xr.Dataset | None:
+    """Get the reference climatology dataset for the variable and season.
+
+    If the reference climatatology does not exist or could not be found, it
+    will be considered a model-only run.
```
Comment (tomvothecoder) [inline e3sm_diags/driver/lat_lon_driver.py:449]:
```suggestion
    If the reference climatatology does not exist or could not be found, it
    will be considered a model-only run and return `None`.
```

File: e3sm_diags/driver/lat_lon_driver.py [hunk 4]
```diff
@@ -265,10 +440,100 @@ def _run_diags_3d(
...
+        ds_test_region = _apply_land_sea_mask(
+            ds_test,
+            ds_land_sea_mask,
+            var_key,
+            region,  # type: ignore
+            parameter.regrid_tool,
+            parameter.regrid_method,
+        )
+
+    # Subset on a specific region.
```
Comment (tomvothecoder) [inline e3sm_diags/driver/lat_lon_driver.py:526]:
```suggestion
```

**Review by tomvothecoder — COMMENTED** [review]
Misc. findings

**Comment by chengzhuzhang** [comment]
> Whenever you have time, can you move the latest fixed version of this file from NERSC? Thanks!

This one is done!

File: docs/source/dev_guide/using-output-viewer.rst [hunk 5]
```diff
@@ -84,18 +86,17 @@ To create a viewer with the Output Viewer:
...
 
-Using CDP Output Viewer (``cdp_viewer``)
-----------------------------------------
-
-``cdp_viewer`` is a wrapper of the Output Viewer to simplify the API for creating a working viewer.
+Using the Output Viewer
+-----------------------
 
-If `cdp_viewer` is too abstract/limited, you can instead use the Output Viewer to create your own viewer.
+If the `OutputViewer` class is too abstract/limited, you can instead use the
```
Comment (tomvothecoder) [inline docs/source/dev_guide/using-output-viewer.rst:92]:
```suggestion
If the ``OutputViewer`` class is too abstract/limited, you can instead use the
```

File: docs/source/dev_guide/using-output-viewer.rst [hunk 6]
```diff
@@ -84,18 +86,17 @@ To create a viewer with the Output Viewer:
...
+ESGF Output Viewer package directly (`import output_viewer`) to create your own viewer.
```
Comment (tomvothecoder) [inline docs/source/dev_guide/using-output-viewer.rst:93]:
```suggestion
ESGF Output Viewer package directly (``import output_viewer``) to create your own viewer.
```

File: docs/source/index.rst [hunk 7]
```diff
@@ -51,9 +51,6 @@ that:
    sensing, reanalysis and in-situ datasets;
 -  interfaces with diagnostics developed from different E3SM focus
    groups: atmosphere group, coupled simulation group, land group;
--  interacts effectively with the PCMDI's metrics package and the ARM
-   diagnostics package through a unifying framework: `Community
-   Diagnostics Package (CDP) <https://github.com/CDAT/cdp>`_.
```
Comment (tomvothecoder) [inline docs/source/index.rst:56]: e3sm_diags no longer uses CDP

**Comment by tomvothecoder** [comment]
Merging this PR based on my review [comment](https://github.com/E3SM-Project/e3sm_diags/pull/846#pullrequestreview-2334448258) above.

**Pull request merged**